src/chaitanyaKDec19/ClassQuestions/PrimeNumSumTest.java

```diff
@@ -0,0 +1,43 @@
+package chaitanyaKDec19.ClassQuestions;
+
+public class PrimeNumSumTest {
+
+	static int primeNumberTotal = 0;
+	boolean flag = false;
+
+	int checkIfPrime(int num) {
+
+		int i;
+
+		for (i = 2; i <= num / 2; i++) {
+
+			if (num % i == 0) {
+				break;
+			}
+		}
+		if (i == (num / 2 + 1)) {
```

Comment:
you can use flag here as well, so other can understand your logic.

Reply:
Sure, next time I will keep it in mid